### PR TITLE
tweeting boost

### PR DESCRIPTION
**Pull request by joaquintides**

### CODE REVIEW

File: contributor-guide/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 * xref:getting-involved.adoc[]
-* xref:tweeting-boost.adoc[]
+* xref:tweeting.adoc[]
```
Comment (alandefreitas) [inline contributor-guide/modules/ROOT/nav.adoc:3]: I don't know if this was on purpose but I find this position in the nav hierarchy extremely bad.

image

It's very unlikely that the first thing a person learning about how to contribute to Boost wants to learn is how people authorized to tweet will tweet.

I thought that would be last and included as a level 2 section of some level 1 section about "branding", "social" or something like that. 

The way it is, it's completely unrelated to any other information in this documentation component, which is all about code.

File: contributor-guide/modules/ROOT/pages/tweeting.adoc [hunk 2]
```diff
@@ -0,0 +1,73 @@
+= Tweeting
+:navtitle: Tweeting
+
+
+The collection is represented on social media by the https://twitter.com/boost_libraries[@Boost_Libraries]
```
Comment (alandefreitas) [inline contributor-guide/modules/ROOT/pages/tweeting.adoc:5]: Should these external links open in other tabs?

IIRC the syntax is something like:

```
https://chat.asciidoc.org[Discuss AsciiDoc,role=resource,window=_blank]
```

**Review by alandefreitas — CHANGES_REQUESTED** [review]

File: contributor-guide/modules/ROOT/nav.adoc (hunk 1 above)
Comment (PeterTurcan) [reply contributor-guide/modules/ROOT/nav.adoc:3]: To match the User Guide, I suggest a top level node, at the end of the TOC:
Community
    Tweeting

Comment (alandefreitas) [reply contributor-guide/modules/ROOT/nav.adoc:3]: "Community" is a great name.

Comment (joaquintides) [reply contributor-guide/modules/ROOT/nav.adoc:3]: @PeterTurcan if I'm understanding you right, you mean one top-level "Community" with just one child "Tweeting"?

Comment (alandefreitas) [reply contributor-guide/modules/ROOT/nav.adoc:3]: Yes. Just one child but we're going to have more pages that can go there. There's stuff related to the ML and there will be a lot of stuff related to the new website.

**Comment by alandefreitas** [comment]
@joaquintides So is this good to go?

**Comment by joaquintides** [comment]
Yes, this is good to go. I wonder if it's possible to do something equivalent to squashing the PR into one commit (project settings don't allow for squash merging).

**Pull request merged**